packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.types.ts

```diff
@@ -41,6 +41,10 @@ export interface IColorPickerProps {
    */
   alphaSliderHidden?: boolean;
 
+  /**
+  whether require a transparency slider.
```

Comment:
This prop name was also updated since I wrote the comment--`showAlphaAsTransparencySlider` is clearer, but I'd prefer something less verbose if possible.

The issue with the comment formatting still needs to be fixed, and the description needs more detail about what "transparency" means vs "alpha." (And my overarching comment *please discuss changes like this with the team beforehand* certainly still applies.)

Reply:
done.